### ISSUE TITLE
Multiple content scripts

### ISSUE THREAD

**Issue by djgrant** [VasilyShelkov/create-react-extension#34]
Would it be possible to create multiple content scripts?

The use case is needing different permissions on different scripts e.g. one content script is used for post-installation, another for logging a user in, another for the general functionality of the page. 

Happy to raise a PR, with some direction

**Comment by dim0147** [comment]
You can try creating a new folder like contentScript folder with the index.js file, then run `yarn eject or npm run eject` and set up a little in the config to create a new JavaScript file when development and build

**Comment by djgrant** [comment]
Ejecting is certainly an option, but best avoided where possible. Especially in common use-cases like this, where other users would also benefit from it being implemented in core.

**Comment by dim0147** [comment]
Currently, there is no way to add new a new file in the build or devevlopment because all the files and folders are hardcode in config with need to eject to see

**Comment by djgrant** [comment]
Yes, I'm aware it's not possible right now. I'm suggesting it be a feature of CRE and am happy to create a PR if the author would like to see it implemented
